compiler/one-cmds/one-optimize

```diff
@@ -56,6 +56,18 @@ def _get_parser():
         help='fuse BatchNorm op to Transposed Convolution op')
     circle2circle_group.add_argument(
         '--fuse_bcq', action='store_true', help='apply Binary Coded Quantization')
+    circle2circle_group.add_argument(
+        '--fuse_preactivation_batchnorm',
+        action='store_true',
+        help='fuse BatchNorm operators of pre-activations to Convolution op')
+    circle2circle_group.add_argument(
+        '--make_batchnorm_gamma_positive',
+        action='store_true',
+        help="make negative gamma of BatchNorm to a small positive value (1e-10). \
+        Note that this pass can change the execution result of the model. \
+        So, use it only when the impact is known to be acceptable.")
```

Comment:
(optional) you can also use triple quote.

And, there is one thing you should know. `one-optimize` is a python script but it doesn't have `.py` extension, which prevent it from format checker recognizing it as a python script. So, you should manually check the format by `yapf`.
```
$ yapf -i compiler/one-cmds/one-optimize
```

Reply:
Thanks for the tip 👍